fs/walk.ts

```diff
@@ -55,6 +56,7 @@ export interface WalkInfo {
  *
  * Options:
  * - maxDepth?: number;
+ * - includeFiles?: boolean;
```

Comment:
I thinks it's nice to have test cases for `includeFiles: false`.

Reply:
👍